doc/src/manual/modules.md

```diff
@@ -119,25 +119,25 @@ end
 
 There are three important standard modules: Main, Core, and Base.
 
-Main is the top-level module, and Julia starts with Main set as the current module.  Variables
-defined at the prompt go in Main, and `varinfo()` lists variables in Main.
+Main is the top-level module, and Julia starts with Main set as the current module. Variables
+defined at the prompt go in Main, and [`varinfo()`](@ref) lists variables in Main.
 
 Core contains all identifiers considered "built in" to the language, i.e. part of the core language
 and not libraries. Every module implicitly specifies `using Core`, since you can't do anything
 without those definitions.
 
-Base is a module that contains basic functionality (the contents of base/). All modules implicitly contain `using Base`,
+Base is a module that contains basic functionality (the contents of `base/`). All modules implicitly contain `using Base`,
 since this is needed in the vast majority of cases.
 
 ### Default top-level definitions and bare modules
 
 In addition to `using Base`, modules also automatically contain
-definitions of the `eval` and `include` functions,
+definitions of the [`eval`](@ref) and [`include`](@ref) functions,
```

Comment:
While you are at it; I think this needs to be corrected for https://github.com/JuliaLang/julia/pull/27098